src/controllers/commentController.js

```diff
@@ -52,4 +54,27 @@ export default class CommentController {
       return Response.errorResponse(res, 500, res.__('server error'));
     }
   }
+
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @param  {object} next
+   * @return {object} delete comment
+   */
+  static async deleteComment(req, res) {
+    try {
+      const { commentId } = req.body;
```

Comment:
hey @Baraka-Mugisha could we use the req.params to contain the commentId rather than putting it in req.body

Reply:
That is an interesting idea